src/main/java/common/AbstractSteps.java

```diff
@@ -0,0 +1,25 @@
+package common;
```

Comment:
please move to package src/test/java/steps

Reply:
Moved